packages/nuxi/src/commands/dev.ts

```diff
@@ -158,6 +158,11 @@ export default defineNuxtCommand({
 
     await load(false)
 
+    const distWatcher = chokidar.watch(resolve(rootDir, '.nuxt/dist'), { ignoreInitial: true })
```

Comment:
I think we can use nuxt.options.buildDir + dist

Reply:
yes :+1: